### ISSUE TITLE
Multiple onComplete generates for each refresh.

### ISSUE THREAD

**Issue by sakthisaravanan** [antony/grails-ajax-uploader#3]
Hi,
    I run my application on firefox6.0. When i refresh the browser it generates more oncomplete function. Check it out.

 <script type="text/javascript">

```
        var au_yourUploaderId = new qq.FileUploader({
        element: document.getElementById('au-yourUploaderId'),
        action: '/ImageCrop/ajaxUpload/upload'
    ,
                multiple: false
            ,
```

onComplete: function(id, fileName, responseJSON) { 
    $('#picture').attr('src',photopath+responseJSON.file);
    $('#cropimage').attr('value',responseJSON.filefullpath);

 $('#picture').Jcrop({
            aspectRatio: 1,
            setSelect: [0, 0, 100, 100],
            minSize: [100, 100],
            maxSize: [100, 100],
            onSelect: updateCoords,
            onChange: updateCoords,
            allowResize: false

```
    });
```

 },
onComplete: function(id, fileName, responseJSON) { 
    $('#picture').attr('src',photopath+responseJSON.file);
    $('#cropimage').attr('value',responseJSON.filefullpath);

 $('#picture').Jcrop({
            aspectRatio: 1,
            setSelect: [0, 0, 100, 100],
            minSize: [100, 100],
            maxSize: [100, 100],
            onSelect: updateCoords,
            onChange: updateCoords,
            allowResize: false

```
    });
```

 }

</script>

**Comment by antony** [comment]
Hi There,

Interesting! I think I know why this might be. I'll see if I can figure out how to fix it!

Thanks for the report,
Antony

**Comment by antony** [comment]
Hi There,

I've fixed this issue in version 0.4 - the taglib was maintaining state between calls from the same controller, so I have now made it reset the two streams which are used for html and javascript content after every render.

Let me know if this works for you.

Thanks for the report!

Cheers,
Antony
